aioshelly/block_device/coap.py

```diff
@@ -134,10 +161,14 @@ def __init__(self, message_received: Callable | None = None) -> None:
         self.subscriptions: dict[str, Callable] = {}
         self.transport: asyncio.DatagramTransport | None = None
 
-    async def initialize(self, socket_port: int = 5683) -> None:
+    async def initialize(
+        self,
+        socket_ips: list[IPv4Address] | None = None,
+        socket_port: int = DEFAULT_COAP_PORT,
+    ) -> None:
```

Comment:
With this change we are creating a breaking API change for `aioshelly` users and if we want to use this for older versions we won't be able too. if we add the `socket_ips` as a last parameter there are no breaking changes
```suggestion
    async def initialize(
        self,
        socket_port: int = DEFAULT_COAP_PORT,
        socket_ips: list[IPv4Address] | None = None,
    ) -> None:
```

Reply:
Good point Shay, let me switch them!